Resources/Prototypes/Corvax/Entities/Objects/Materials/Sheets/other.yml

```
@@ -0,0 +1,32 @@
+- type: entity
+  parent: Paper
+  id: SheetPrinter
+  suffix: Full
+  components:
+  - type: Material
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)